### ISSUE TITLE
Ctrlv issue

### ISSUE THREAD

**Issue by ethxrnity** [DeadBread76/Raid-Toolbox#153]
I cant use ctrlv in the program, is there a problem with code or how can i fix that?

**Comment by hidden-r4d** [comment]
Are you sure ctrl+v works in other programs? Are you sure you have text copied to you clipboard?

**Comment by ethxrnity** [comment]
> Are you sure ctrl+v works in other programs? Are you sure you have text copied to you clipboard?

Yes, it works fine for example on Discord, LoL client or Facebook. And yes i'm.

**Comment by DeadBread76** [comment]
most likely a pysimplegui bug

**Comment by ethxrnity** [comment]
Is there a way to fix it?
